benchmark/README.md

```diff
@@ -22,4 +22,7 @@ Notes:
 | `zq` | `discovery` | zng | zng | 2.23 | 3.67 | 0.10 |
 | `jq` | `analytics` | ndjson | ndjson | 35.28 | 36.50 | 2.72 |
 | `jq` | `search` | ndjson | ndjson | 26.09 | 27.48 | 2.61 |
-| `jq` | `discovery` | ndjson | ndjson | 32.59 | 33.76 | 2.80 |
\ No newline at end of file
+| `jq` | `discovery` | ndjson | ndjson | 32.59 | 33.76 | 2.80 |
+| `elastic search` | `search: find` | ndjson | elastic index | 12 | n/a | n/a |
+| `elastic search` | `search: find and sort` | ndjson | elastic index | 34 | n/a | n/a |
+| `elastic search` | `search: find, sort, return top n` | ndjson | elastic index | 7 | n/a | n/a |
```

Comment:
I think the input and output formats are backwards here? The queries are issued over elastic indexes and output NDJSON results.

Also, are these numbers seconds or milliseconds? If they're milliseconds, please represent them as seconds instead (e.g., 0.012) to be consistent with the rest of the table. Thanks!